.gitlab-ci.yml

```diff
@@ -38,6 +38,7 @@ variables:
 .global_trigger_compat_tests: &global_trigger_compat_tests
   rules:
     - if: $CI_COMMIT_BRANCH == "master"
+    - if: $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == "master"
```

Comment:
maybe the test should be on the source branch being `develop`, no?

Reply:
I wanted to be a bit more explicit in saying "anything that gets to master needs to pass here", which may be stuff from other branches than develop.